containers-orchestration/managed-kubernetes/use-image-from-registry/README.md

```diff
@@ -0,0 +1,57 @@
+## Use image from a MRS with OVHcloud Managed Kubernetes
+
+### Setup
+
+  - Deploy a MKS cluster
+  - Install the kubectl CLI
+  - Install the kustomize CLI
+
+### Demo
+
+  - set the environment variables:
+
+```bash
+# OVHcloud provider needed keys
+export PRIVATE_REGISTRY_URL="xxx"
+export PRIVATE_REGISTRY_USER="xxx"
+export PRIVATE_REGISTRY_PASSWORD="xxx"
+export PRIVATE_REGISTRY_PROJECT="xxx"
+export PRIVATE_REGISTRY_URL_WITHOUT_SCHEME=${PRIVATE_REGISTRY_URL#*//}
```

Comment:
I don't know if the characters `#*//` are normal or not?

Reply:
Yes the characters are normal, in fact it's a trick :-) 

If you have a text with a URL, example "".
If you want the URL but without the "https://", instead of doing a awk, or a sed you can simple execute `THE_TEXT#*//`

example:

```
$  PRIVATE_REGISTRY_URL="https://abcdefghi.x1.gra9.container-registry.ovh.net"
$ PRIVATE_REGISTRY_URL_WITHOUT_SCHEME=${PRIVATE_REGISTRY_URL#*//}

$ echo $PRIVATE_REGISTRY_URL
https://abcdefghi.x1.gra9.container-registry.ovh.net

$ echo $PRIVATE_REGISTRY_URL_WITHOUT_SCHEME
abcdefghi.x1.gra9.container-registry.ovh.net
```

Magic! :-)